cilium-cli/install/autodetect.go

```diff
@@ -74,18 +74,27 @@ func getClusterName(helmValues map[string]any) string {
 	return clusterName
 }
 
-func trimEKSClusterARN(fullARN string) string {
+func trimEKSClusterName(identifier string) string {
```

Comment:
Now that you understand this function more than anyone else in the world, how about a quick docblock :-)